### ISSUE TITLE
Possible to include a global SASS-File?

### ISSUE THREAD

**Issue by mathishuettl** [leinelissen/laravel-mix-react-css-modules#19]
Hello,

is it possible to include a global .scss file (with variables and mixins) and use the variables/mixins in a styles.module.scss file?

**Comment by leinelissen** [comment]
Yes! You can create a global file and a set of mix-ins, and then import them in each individual CSS Module file using the SASS-standard `@import` rule (see https://sass-lang.com/documentation/at-rules/import).
